base/NCIO.F90

```diff
@@ -2701,12 +2711,18 @@ subroutine MAPL_VarWriteNCpar_R8_2d(formatter, name, A, ARRDES, lev, offset2, oC
           _VERIFY(STATUS)
        endif
 
+       t0 = mpi_wtime()
        call mpi_gatherv( a, size(a), MPI_DOUBLE_PRECISION, recvbuf, recvcounts, displs, &
                          MPI_DOUBLE_PRECISION, 0, arrdes%iogathercomm, status )
        _VERIFY(STATUS)
+       t1 = mpi_wtime()
+       call mpi_allreduce(t1-t0,maxtime,1, MPI_REAL, MPI_MAX, arrdes%iogathercomm, status )
+       _VERIFY(STATUS)
 
        if(myiorank==0) then
 
+         !write(*,'(A,E11.4)') 'Max time spent in R8_2d mpi_gatherv (seconds) ', maxtime
```

Comment:
This should be deleted or turned into a logger call.

Reply:
I deleted it